.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,24 @@
 <!-- You can remove tags that do not apply. -->
-Resolves: # <!-- Refer an issue of this repository that this PR fixes -->
-See: <url> <!-- Refer to any external resource, like a PR, document or discussion -->
-Depends-on: <url> <!-- This change depends on a PR to get merged/deployed first -->
-Change-type: major|minor|patch <!-- The change type of this PR -->
+Resolves: # <!-- Refer an issue of this repository that this PR fixes -->  
+Change-type: major|minor|patch <!-- See https://semver.org/ -->  
+Depends-on: <url> <!-- This change depends on a PR to get merged/deployed first -->  
+See: <url> <!-- Refer to any external resource, like a PR, document or discussion -->  
 
 ---
-##### Contributor checklist
```

Comment:
This Contributor Checklist was put together years ago. I found it was a bit odd, especially because a "checklist" is usually a list of things that someone should do (before submitting the PR), usually starting with imperative verbs:
```
[ ] Read something
[ ] Write test code
[ ] Run something
```
But _"Introduces security considerations"_ is not something someone should _do_. So I replaced the checklist with some "guidance paragraphs". Suggestions are welcome.

This change is incidental and piggybacked on this PR, but it's in a separate commit. If it becomes contentious or something like that, I'll move it to a separate PR.

Reply:
Yeah I agree.  I wonder if that was introduced in response to something bad that happened, or just for good measure.